test/expect/TestJit.test_assign_traces.expect

```diff
@@ -1,8 +1,8 @@
 graph(%0 : Double(10, 10)
       -------- stage 1 --------
-      %1 : Double(10, 10!)) {
+      %3 : Double(10, 10!)) {
   %2 : Double(10, 10) = ^MyFn()(%0)
   ---------------- stage 1 ----------------
-  %3 : Double(10, 10) = aten::mul(%2, %1)
-  return (%2, %3);
+  %4 : Double(10, 10) = aten::mul(%2, %3)
```

Comment:
These got changed because an extra graph copy was performed. It would be better to just fix the pass to avoid t his copy. Then these changes should go away,

Reply:
Are you sure it's an extra copy being performed in my change? It seems like the extra copy existed before my change and my change got rid of it. It looks like in my case the copy was *not* performed and the unique identifiers were not rematerialized

Reply:
Ok, new hypothesis: Canonicalize pass is broken